src/CommunityToolkit.Maui.Markup.UnitTests/CommunityToolkit.Maui.Markup.UnitTests.csproj

```diff
@@ -2,18 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetVersion)</TargetFramework>
+    <OutputType>Exe</OutputType>
     <IsPackable>false</IsPackable>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GF</CompilerGeneratedFilesOutputPath>
+    <RootNamespace>CommunityToolkit.Maui.Markup.UnitTests</RootNamespace>
+    <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
```

Comment:
I think this one is specific for xunit. Not sure if it's the same for nunit

Reply:
Got it - Let's find out! I'll remove it and re-run the CI pipeline 👍